content/about/releases.en.md

```diff
@@ -11,4 +11,6 @@ After six months, odd-numbered releases (9, 11, etc.) become unsupported, and ev
 _LTS_ release status is "long-term support", which typically guarantees that critical bugs will be fixed for a total of 30 months.
```

Comment:
This is already explained on this line

Reply:
I will remove it from this part

Reply:
Yes I have to remove it from the text it's an oversight on my part. the idea is to highlight it